client/src/components/common/P5SimModal.tsx

```diff
@@ -0,0 +1,95 @@
+import { Modal } from "antd";
+import { Rnd } from "react-rnd";
+import { useState } from "react";
+import { Sim } from "../../types";
+
+/**
+ * This component is exclusively for displaying a p5 sketch iframe
+ * Since the size of the iframe is uncertain, the size of the modal box
+ * gets changed as we resize the sim
+ */
+
+/**
+ * Accepts all the props of Modal in antd along with a
+ * sim object which is of the form { owner:string, id: string }
+ */
+
+const P5SimModal = (props: any) => {
+  let sim: Sim;
+
+  const { onCancel, onOk } = props;
+
+  sim = props.sim;
+
+  const [size, setSize] = useState({ width: 640, height: 360 });
+
+  const handleResize = (
+    _e: any,
+    _direction: any,
+    ref: any,
+    _delta: any,
+    _position: any
+  ) => {
+    setSize({
+      width: parseInt(ref.style.width),
+      height: parseInt(ref.style.height),
+    });
+  };
+
+  let { width, height } = size;
```

Comment:
Is it possible to use variable name dimensions instead of size?Since it is not a single value it has both width and height.

Reply:
The term size is the one used by the library, so I thought I'd use the same